airbyte-integrations/connectors/source-surveymonkey/unit_tests/unit_test.py

```diff
@@ -0,0 +1,63 @@
+#
```

Comment:
please rename this file to `test_streams.py` or something.
it is always good to name your test as action, also `unit` in the name is too broad